sw/vendor/riscv_compliance.vendor.hjson

```diff
@@ -0,0 +1,26 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "riscv_compliance",
+  target_dir: "riscv_compliance",
+
+  upstream: {
+    url: "https://github.com/riscv/riscv-compliance.git",
+    rev: "master",
+  },
```

Comment:
Now that you've merged your changes into the `ot` branch of https://github.com/lowRISC/riscv-compliance, please add `patch_repo` section like this:

```hjson
  patch_repo: {
    url: "https://github.com/lowRISC/riscv-compliance.git",
    rev_base: "master",
    rev_patched: "ot",
  },
```

You can then re-run the vendor script with ```--refresh-patches --commit --verbose` to generate a commit with the patches and the upstream sources. 

This should replace the `[vendor] Vendor in riscv compliance code ` and `[test] Add riscv_compliance patch` commits of this PR.

Reply:
yep will do this today. I just didnt get a chance to finish this part when I took off.

Reply:
for this portion, can i do it as a separate PR? It seems like the mainline riscv_compliance has updated, so the patches in the patch repo do not apply correctly anymore.

I'm assuming what I need to do now is first update the lowrisc port to tip of tree, make my changes on top of that, and then try the vendor flow again.  Is that right?

Or alternatively if I could modify the vendor script to checkout a specific version instead of always using head of tree.

Reply:
you can and should push directly to our fork repository (no need for PRs there). Rebase `ot` on top of the upstream master, and then push the upstream master and ot both to the forked repository. See also "How to manage patches in a Git repository" in https://docs.opentitan.org/doc/ug/vendor_hw/, or let me know if you're lost and I'll do it.

Reply:
i have been reading that, but i guess i'm not sure what you mean by push directly (sorry not a git expert). 

Let me give it a try later, right now I need to figure out why CI isn't working again even though it worked last time.

Reply:
alright..so i think i did that..but it seems with the latest updates, riscv_compliance is now failing some tests.

I really prefer to get this one in for now.  Then we can go back and debug the issues and also include the patch repo.  Is that okay with you?